### PR TITLE
OBSDOCS-933: Add documentation for forward connector

### PR DESCRIPTION
**Pull request by max-cx**
:warning: This PR is tied to a product release date, which is currently set to **June 5** (Wednesday).

- If the merge review gets delayed, you're welcome to ping `mleonov` on Slack.
- If you have any questions, ping `mleonov` on Slack.

Edit of https://github.com/openshift/openshift-docs/pull/73628

Signed-off-by: Israel Blancas <iblancasa@gmail.com>

Version(s):

4.12, 4.13, 4.14, 4.15, 4.16

Issue:

https://issues.redhat.com/browse/OBSDOCS-933
https://issues.redhat.com/browse/TRACING-4066

Link to docs preview:

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/otel-collector-components.adoc [hunk 1]
```diff
@@ -740,6 +740,46 @@ The Kafka exporter exports logs, metrics, and traces to Kafka. This exporter use
...
+      processors: []
+      exporters: [forward]
+    traces/regionb:
+      receivers: [jaeger]
+      processors: []
+      exporters: [forward]
+    traces:
+      receivers: [forward]
+      processors: [batch]
+      exporters: [bar]
```
Comment (IshwarKanse) [inline modules/otel-collector-components.adoc:780]: The exporter should be otlp here.

File: modules/otel-collector-components.adoc [hunk 2]
```diff
@@ -740,6 +740,46 @@ The Kafka exporter exports logs, metrics, and traces to Kafka. This exporter use
...
+    protocols:
+      grpc:
+  jaeger:
+    protocols:
+      grpc:
+processors:
+  batch:
+exporters:
+  otlp:
+    endpoint: tempo-ingester:4317
```
Comment (IshwarKanse) [inline modules/otel-collector-components.adoc:762]: We ingest the traces to distributor. We can change it to tempo-simplest-distributor:4317 in this docs example.

Comment (max-cx) [reply modules/otel-collector-components.adoc:762]: Fixed. Thank you.

File: modules/otel-collector-components.adoc (hunk 1 above)
Comment (max-cx) [reply modules/otel-collector-components.adoc:780]: Changed. Thank you.

**Review by JoeAldinger — COMMENTED** [review]
/lgtm

**Review by iblancasa — APPROVED** [review]

**Comment by max-cx** [comment]
Closing this PR because I cherry-picked its commit into https://github.com/openshift/openshift-docs/pull/76340.

**Pull request merged**